torch/onnx/symbolic_opset11.py

```diff
@@ -115,6 +115,7 @@ def __interpolate(g, input, size, scale_factor, mode, align_corners):
             size = unsqueeze(g, size, 0)
             size = [size for i in range(input.type().dim() - 2)]
             size = g.op("Concat", *size, axis_i=0)
+        size = g.op("Cast", size, to_i=sym_help.cast_pytorch_to_onnx['Long'])
```

Comment:
although this won't hurt, I am wondering why we need the new cast?

Reply:
Scripting seems to have some bug related to types. 
I am not sure why it is happening, but "size" is a float in keypointrcnn, and fails later because of a type mismatch. 
We need to understand the real issue and fix it, but this does the trick for now to support keypointrcnn.